onset_maproom/tests/test_agronomy.py

```diff
@@ -317,19 +317,27 @@ def test_soil_plant_water_balance_with_et_crop():
         rho=None,
         runoff=agronomy.weekly_api_runoff(precip_sample()),
     )
-    print(wat_bal.soil_moisture)
 
     assert (wat_bal.et_crop == wat_bal.et_crop_red).all()
-    expected = [[[55.81081862, 56.74815773]],
-       [[57.01759313, 58.28147063]],
-       [[58.09657863, 59.71744128]],
-       [[60.        , 60.        ]],
-       [[59.35689346, 59.72076937]],
-       [[58.92301271, 59.68844135]],
-       [[57.8169261 , 58.98686702]],
-       [[60.        , 60.        ]],
-       [[60.        , 60.        ]],
-       [[60.        , 60.        ]]]
+    expected = [[[58.28297711,  0.        ]],
+
+       [[60.        ,  0.        ]],
+
+       [[60.        ,  0.        ]],
+
+       [[60.        ,  6.34475728]],
+
+       [[60.        ,  4.69855057]],
+
+       [[60.        ,  3.27655007]],
+
+       [[59.68250804,  1.27869775]],
+
+       [[60.        , 13.15848481]],
+
+       [[60.        , 14.09643733]],
+
+       [[60.        , 14.15885817]]]
```

Comment:
Was it black that added all the blank lines?

Reply:
no... sorry... It's just a copy and past from terminal, from printing the result to be checked in test for reproducibility. I will make nicer.